README.md

```diff
@@ -1,11 +1,11 @@
 # Orchestra
 
 [![PyPI version](https://badge.fury.io/py/orchestra.svg?maxAge=2592000)](https://badge.fury.io/py/orchestra)
-[![Documentation Status](https://readthedocs.org/projects/orchestra/badge/?version=latest)](http://orchestra.readthedocs.org/en/stable/?badge=latest)
+[![Documentation status](https://readthedocs.org/projects/orchestra/badge/?version=latest)](http://orchestra.readthedocs.org/en/stable/?badge=latest)
 [![Circle CI](https://circleci.com/gh/b12io/orchestra.svg?maxAge=2592000&style=shield)](https://circleci.com/gh/b12io/orchestra)
-[![Coverage Status](https://coveralls.io/repos/github/b12io/orchestra/badge.svg?maxAge=2592000&branch=master)](https://coveralls.io/github/b12io/orchestra?branch=master)
-[![Apache License](https://img.shields.io/pypi/l/orchestra.svg?maxAge=2592000)](https://github.com/b12io/orchestra/blob/master/LICENSE)
-[![Gitter](https://img.shields.io/gitter/room/nwjs/nw.js.svg?maxAge=2592000)](https://gitter.im/b12io/orchestra)
+[![Coverage status](https://coveralls.io/repos/github/b12io/orchestra/badge.svg?maxAge=2592000&branch=master)](https://coveralls.io/github/b12io/orchestra?branch=master)
+[![Apache license](https://img.shields.io/pypi/l/orchestra.svg?maxAge=2592000)](https://github.com/b12io/orchestra/blob/master/LICENSE)
+[![Discussion forum](https://img.shields.io/badge/forum-GH%20Discussions-brightgreen)](https://github.com/b12io/orchestra/discussions)
```

Comment:
such sentence-case wao.

Reply:
It looks like it adds to alt text, which checks out to me :)

![image](https://user-images.githubusercontent.com/57725/108903763-af13b200-75eb-11eb-864b-9e3c7c86bd4f.png)